ANMN/common/incoming_handler.sh

```diff
@@ -60,11 +58,13 @@ handle_netcdf() {
 handle_zip() {
     local zipfile=$1; shift
 
+    local unzip_dir=`mktemp -d --tmpdir ${JOB_NAME}_XXXXX`
+    chmod 755 $unzip_dir  # make directory world readable
+
     # unzip the file
-    log_info "Unzipping '$zipfile' to $UNZIP_DIR"
-    mkdir -pv $UNZIP_DIR
-    local extracted_files=`mktemp`
-    unzip_file $zipfile $UNZIP_DIR $extracted_files || file_error "Failed to unzip"
+    log_info "Unzipping '$zipfile' to $unzip_dir"
+    local extracted_files=`mktemp --tmpdir=$unzip_dir`
+    unzip_file $zipfile $unzip_dir $extracted_files || file_error "Failed to unzip"
```

Comment:
Thanks for the comment, but I'm not sure what you mean. So far I've only heard the term "manifest file" used in the context of bulk indexing, and that is not what this is. It's simply a list of files that were extracted from the zipfile. So I think `extracted_files` is a pretty good name for it. I could call it `extracted_files_list` if that makes it clearer?